platform/ui/src/components/MeasurementTable/MeasurementTable.tsx

```diff
@@ -1,13 +1,31 @@
 import React from 'react';
+import { ServicesManager } from '@ohif/core';
 import PropTypes from 'prop-types';
 import { useTranslation } from 'react-i18next';
 
 import MeasurementItem from './MeasurementItem';
 
-const MeasurementTable = ({ data, title, onClick, onEdit }) => {
+const MeasurementTable = ({
+  data,
+  title,
+  onClick,
+  onEdit,
+  servicesManager,
+}) => {
+  servicesManager = servicesManager as ServicesManager;
+  const { customizationService } = servicesManager.services;
```

Comment:
So the idea was UI not get touched with the ohif core (services, managers, etc.), can we achieve the same thing by doing what you are doing with customizationService at the panel level?

Reply:
That is what I'm mostly trying to achieve, but the basic data needs to be available to the panel in a format it can detect, so that does require some changes to the surrounding code.  I can pull out all the changes related to actually displaying the data if it exists, although that only gets invoked if you are already using data that has the extra information.  The parsers have parsed it all along, just that it has been ignored or thrown out.

Reply:
The UI should be unchanged unless you specifically add customizations - that is, it looks identical until you start adding customizations.